README.md

```diff
@@ -117,55 +113,44 @@ Don't do: `Nri/Leaderboard/Main.elm` - the filename `Main.elm` is reserved for e
 Our Elm apps generally take this form:
 
 - Main.elm
-- Model.elm
-- Update.elm
-- View.elm
-- Flags.elm
+    - `type alias Flags = { ...a record that directly corresponds to the JSON page data... }`
+    - `flagsDecoder : Json.Decode.Decoder Flags`
+    - `type alias Model = { ...a record with fields... }`
+    - `init : Flags -> (Model, Cmd Msg)`
+    - `type Msg = ... variants for each possible message ...`
+    - `update : Msg -> Model -> (Model, Cmd Msg)`
+    - `view : Model -> Html Msg`
 
-Inside **`Model.elm`**, we contain the actual model for the view state of our program. Note that we generally don't include non-view state inside here, preferring to instead generalize things away from the view where possible. For example, we might have a record with a list of assignments in our `Model` file, but the assignment type itself would be in a module called `Data.Assignment`.
+Inside **`Model`**, we contain the actual model for the view state of our program. Note that we generally don't include non-view state inside here, preferring to instead generalize things away from the view where possible. For example, we might have a record with a list of assignments in our `Model` file, but the assignment type itself would be in a module called `Data.Assignment`.
 
-**`Update.elm`** contains our update code. This includes the `Msg` types for our view. Inside here most of our business logic lives.
+**`Msg`, `update`** contains our update code. Inside here most of our business logic lives.
 
-Inside **`View.elm`**, we define the view for our model and set up any event handlers we need.
+Inside **`view`**, we define the view for our model and set up any event handlers we need.
 
-**`Flags.elm`** contains a decoder for the flags of the app. We aim to keep our decoders basic and so decode into a special `Flags` type that mirrors the structure of the raw JSON instead of the structure of the `Model` type. The `Flags` and `Model` modules should not depend on each other.
+**`Flags`, `flagsDecoder`** is a decoder for the flags of the app. We aim to keep our decoders basic and so decode into a special `Flags` type that mirrors the structure of the raw JSON instead of the structure of the `Model` type. The `Flags` and `Model` modules should not depend on each other.
```

Comment:
I think calling this `flagsDecoder` is a change. In the monolith, I searched for `decoder :` and `flagsDecoder :`. There are only 14 results for `flagsDecoder` (and too of those are from `Nri.Program`).

I wonder if we should talk about changing to `flagsDecoder` in a separate convo from the modules convo. But also, not a big deal.

Reply:
I'm fine with `decoder` here if that's what's in use -- I didn't actually check; I just guessed that it was being called `flagsDecoder` :)

Reply:
updated